src/js/data/CoinInfo.js

```diff
@@ -237,7 +237,7 @@ const parseBitcoinNetworksJson = (json: JSON): void => {
 
             // used in backend ?
             blocks: Math.round(coin.blocktime_seconds / 60),
-            decimals: 8,
+            decimals: coin.divisibility,
```

Comment:
I've opened issue about [names inconsistency](https://github.com/trezor/trezor-firmware/issues/743) and it will be changed